docs/change-controlled-docs.rst

```diff
@@ -0,0 +1,233 @@
+.. _lsst-texmf: https://lsst-texmf.lsst.io
+.. _LPM-19: https://ls.st/LPM-19
+.. _LPM-51: https://ls.st/LPM-51
+.. _GitHub: https://github.com/lsst
+.. _LDM-294: https://ls.st/LDM-294
+
+.. _ccd-regular:
+
+Regular development
+===================
+
+Develop change-controlled documents using :doc:`DM's standard workflow <../processes/workflow>`.
+That is:
+
+1. Create a ticket branch from ``master``.
+2. Work on the ticket branch.
+3. Have the ticket peer-reviewed.
+4. Rebase and merge the ticket branch to ``master``.
+
+Merging to ``master`` does not denote acceptance by a CCB.
+Instead, it adds to the changeset that will be included in the next CCB review.
+
+.. figure:: ccd-develop.svg
+   :name: fig-ccd-develop
+   :align: center
+   :target: ccd-develop.svg
+   :alt: Git commit tree showing two ticket branches merged to master.
+
+   Example of Git branches during document development.
+   Staff write and edit the document on individual ticket branches that they merge to master, following DM's regular code workflow.
+
+.. _ccd-submit:
+
+Submitting a document to the CCB
+================================
+
+Establishing a new baselined version of the document begins with submitting it to the CCB.
+The release manager is responsible for orchestrating this process on behalf of DM.
+
+1. Choose a commit on ``master`` where the document's content is ready for CCB review.
+2. Create a DocuShare tag at the commit.
+   DocuShare tags are formatted as ``docushare-vN``, where ``N`` is the next-available document version number in DocuShare.
+3. Upload the PDF built by CI for the ``docushare-vN`` tag to DocuShare.
+   By definition, this uploaded document should become version ``N`` in DocuShare.
+4. For LDM documents, create a DM RFC.
+   For project documents (LSE), create an LCR.
+5. Once the RFC or LCR number is known, create a branch from the ``docushare-vN`` tag named ``tickets/RFC-N`` or ``tickets/LCR-N``.
+   This release branch initially has no new commits on it.
+
+.. figure:: ccd-submit.svg
+   :name: fig-ccd-submit
+   :align: center
+   :target: ccd-submit.svg
+   :alt: Git commit tree showing a docushare-v1 tag added to the head of the master branch.
+
+   Example Git tree showing document submission to the CCB.
+   The release manager creates a ``docushare-v1`` tag from the head of the ``master`` branch, since ``1`` is the next-available version number for the document in DocuShare.
+   The release manager submits the PDF built from the tag build to the CCB (as an RFC or an LCR).
+   In this case, the submission is ``RFC-1`` so the release manager creates the ``tickets/RFC-1`` release branch from ``docushare-v1`` tag.
+
+.. _ccd-edit:
+
+Addressing CCB feedback
+=======================
+
+The CCB may request amendments before the document can be baselined.
+
+1. Create a ticket branch from the ``tickets/RFC-N`` or ``tickets/LCR-N`` release branch.
+2. Commit amendments to that ticket branch and peer review.
```

Comment:
Do we need peer review if the CCB have requested a change? The CCB will be about to review the change anyhow. When dealing with major feedback that was not clear peer review would be useful but are we mandating it?